deltachat-ios/Controller/EditSettingsController.swift

```diff
@@ -31,14 +37,24 @@ class EditSettingsController: UITableViewController {
         return cell
     }()
 
-    lazy var accountSettingsCell: UITableViewCell = {
+    private lazy var accountSettingsCell: UITableViewCell = {
         let cell = UITableViewCell(style: .value1, reuseIdentifier: nil)
         cell.textLabel?.text = String.localized("pref_password_and_account_settings")
         cell.accessoryType = .disclosureIndicator
-        cell.accessibilityIdentifier = "accountSettingsCell"
+        cell.tag = tagAccountSettingsCell
         return cell
     }()
 
+
+    private lazy var pictureAndNameCell: AvatarEditTextCell = {
+        let contact = DcContact(id: Int(DC_CONTACT_ID_SELF))
+        let cell = AvatarEditTextCell(context: dcContext, defaultImage: defaultImage)
+        cell.inputField.text = contact.displayName
```

Comment:
this is wrong.

contact.displayName is typically "Me" as displayed in lists etc. when setting this to the inputField, recipient will get the text "Me" as the sender name.

to get/set the configured displayname that is actually sent to others, use dcContext.get|setConfig("displayname"). it is okay, if this string is empty for "no name set"

Reply:
thanks! fixed